.github/workflows/tests.yaml

```diff
@@ -0,0 +1,35 @@
+name: R2Inference CI
+
+on:
+  push:
+    branches:
+      - master
+      - dev-*
+      - feature/ci-experiments
+  pull_request:
+    branches:
+      - dev-*
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    container: ridgerun/r2inference:v0.1.0
+    env: 
+      PREFIX: /usr/lib/x86_64-linux-gnu/
```

Comment:
Prefix is unneeded in this version. Also, this prefix would be wrong. The prefix is the portion before `lib`, `include`, `bin`, etc... In this case it would be `PREFIX=/usr`. The variable you're looking for is `LIBDIR=/usr/lib/x86_64-linux-gnu` (which again is not needed on this project.